### DISCUSSION TITLE
Can you "fix" the Wavesurfer to a position? (Possibly bug too)

### DISCUSSION THREAD

**Discussion by Artur-Galstyan** [katspaugh/wavesurfer.js#3303] [Q&A]
Let's say I don't want to have a scrollbar and want the user ONLY to see a section in the middle of the song. 

I know that I can seek to the middle and if `dragToSeek: false` and `hideScrollbar: true` then the user will only see the middle. So far, so good.

But I have a button that opens a `<dialog>`. After closing it, the Wavesurfer *always* jumps to the beginning of the song. That might even be a bug @katspaugh

Question: Can you fix the Wavesurfer to *never* change its current "scroll position"?

**Comment by katspaugh** [comment]
> But I have a button that opens a <dialog>. After closing it, the Wavesurfer always jumps to the beginning of the song. 

Can you elaborate on what this dialog does? Unless you're re-creating or re-rendering wavesurfer, or setting a time, it shouldn't jump anywhere.

**Reply by Artur-Galstyan** [reply]
It actually does nothing. 

```svelte
<div>
	<button
		on:click={() => {
			toggleModal('test111'); //simply opens the modal
		}}
		class="btn"
	>
		Test
	</button>
</div>

<dialog id="test111" class="modal">
	<form method="dialog" class="modal-box w-11/12 max-w-7xl h-[70%] border border-primary">
		<h1>Hi</h1>
	</form>
</dialog>

```

**Reply by katspaugh** [reply]
Looks like you're using some sort of UI framework, so I would double-check that it doesn't re-render the wavesurfer.
